### ISSUE TITLE
Why inputs are scaled using x *= 0.017 on DenseNet implementation?

### ISSUE THREAD

**Issue by plesqui** [keras-team/keras-contrib#493]
Greetings,

I'm trying to use pre-trained weights from DenseNet-169 and I need to figure out the right normalization for my input data. I see, on the preprocess_input function that inputs are first normalized to -1, +1 (for tf) and subsequently they are scaled by multiplying them by 0.017.

Does anyone know the rationale for this additional scaling? 

Thank you in advance!
